include/mbedtls/ssl.h

```diff
@@ -5065,19 +5065,31 @@ int mbedtls_ssl_close_notify(mbedtls_ssl_context *ssl);
 
 #if defined(MBEDTLS_SSL_SRV_C)
 /**
- * \brief          Read at most 'len' application data bytes while performing
- *                 the handshake (early data).
+ * \brief          Read at most 'len' bytes of early data
+ *
+ * \note           This API is server specific.
+ *
+ * \note           Early data is defined in the TLS 1.3 specification, RFC 8446.
```

Comment:
If Early Data has different security properties from normal data (when it comes to replay for example) this should be mentioned here.